crypto/keys/pem/pem_test.go

```diff
@@ -91,10 +91,10 @@ func TestFromProto(t *testing.T) {
 		},
 	} {
 		signer, err := FromProto(test.keyProto)
-		if gotErr := err != nil; gotErr != test.wantErr {
+		if (err != nil) != test.wantErr {
 			t.Errorf("%v: FromProto(%#v) = (_, %q), want (_, nil)", test.desc, test.keyProto, err)
 			continue
-		} else if gotErr {
+		} else if err != nil {
```

Comment:
Can get rid of the `else` now that there's no shared `gotErr` var.

Reply:
Reverted.